indico/modules/events/templates/emails/move_request_closure.txt

```diff
@@ -0,0 +1,33 @@
+{% extends 'emails/base.txt' %}
+
+{% block subject -%}
+    Event move request update
+{%- endblock %}
+
+{% block header -%}{%- endblock %}
+
+{% block body -%}
+    {%- filter dedent -%}
+        {% if accept %}
+            The following events have been moved:
+        {% else %}
+            Move requests of the following events have been rejected:
+        {% endif %}
+
+        {% filter underline %}Events{% endfilter %}
+        {%- for evt in events %}
+            {{ evt.title }} ({{ evt.external_url }}) {# -#}
+        {% endfor %}
+
+        {% if not accept %}
```

Comment:
We probably need to collapse some of the space here so there's an equal amount of blank lines before the signature regardless of whether there's a reason or not